packages/svelte-ux/src/lib/utils/date.ts

```diff
@@ -549,11 +575,181 @@ export function formatISODate(
   return formatISO(date, { representation });
 }
 
+export enum DateToken {
```

Comment:
Highlight: Currently handle tokens. (you can always escape and use IntlOptions)

Reply:
Most of my code is camelCase, but the more I see snake_case, the more I like it. I know that for consistency, it could be better to go back to camelCase.

I feet it a bit strange with `dayOfMonth2Digit` (because au the 2)
I was also thinking about: `dayOfMonth_2Digit_dd` to give as much info as possible with just the name. like: `meaningFullName_intlInfo_fnsFormatFYI`

Let me know what's your final, though, I'll adapt to it 👍

Reply:
I don’t mind snake case (full disclosure, I wrote a lot of Python about 10 years ago). I also believe the Svelte project uses it (Rich’s preference if I recall). With that said, I’d rather be consistent and maybe one day switch. I do prefer camel case for props (and aligns with DOM apis more). I’m fine with using them for meaningful dividers in this case.

So in short, prefer camel case but can use underscores with enough reason.   Sound good to you?

Reply:
Looks good.  Thanks!